src/api/app/models/bs_request.rb

```diff
@@ -1278,7 +1277,6 @@ def self.extend_relation(source_or_target, requests, roles, package, subprojects
   def self.extend_query_for_maintainer(obj, requests, roles, inner_or)
     if roles.empty? || roles.include?('maintainer')
       names = obj.involved_projects.pluck('name').map { |p| quote(p) }
-      requests = requests.references(:bs_request_actions)
```

Comment:
Not sure if you can remove it here as ``requests`` gets called in ``BsRequest.collection`` (1143) with ``requests = with_actions``, so actually it does join with ``bs_request_actions``